src/main/java/com/google/cloud/teleport/templates/PubsubToAvro.java

```diff
@@ -169,16 +225,25 @@ public static PipelineResult run(Options options) {
     // Create the pipeline
     Pipeline pipeline = Pipeline.create(options);
 
+    ValueProvider<String> inputSubscription = options.getInputSubscription();
```

Comment:
Extra assignment statements inputSubscription, inputTopic can be eliminated

Reply:
👍